js/map.js

```diff
@@ -22,7 +22,11 @@ function selectParcel(lat, lng, instance) {
         'location': latlng
       }, function (results, status) {
         if (instance.data_available) {
-          instance.home_address = results[0].formatted_address;
+          instance.home_address = results[0].formatted_address; // customCalscapeUrl update
+
+          var _url = "https://calscape.org/loc-".concat(lat, ",").concat(lng, "(").concat(instance.home_address, ")/?&poploc=1");
```

Comment:
Any idea what the `poploc=1` does?

Reply:
Sounds like a dance move?

Reply:
Makes the Calscape website not break lol. Tbh not sure beyond that though.